chart/values.global.yaml

```diff
@@ -1,4 +1,7 @@
 fullnameOverride: "llmaz"
 
 backendRuntime:
+  enabled: true
+
+leaderElectEnabled:
   enabled: true
```

Comment:
Seems we can simply add an configuration here, then it works, see example below. The snippet helps people understand how to configure the controller manager.

```
controllerManager:
  manager:
    args:
    - --health-probe-bind-address=:8081
    - --metrics-bind-address=127.0.0.1:8080
    - --leader-elect
  replicas: 1
```

Once we configured here, then run `make helm-install`, the configuration works.